Volume/Views/MainView/MagazinesList.swift

```diff
@@ -0,0 +1,134 @@
+        // TODO: Replace with Magazines request
+        cancellableQuery = Network.shared.publisher(for: GetAllPublicationsQuery())
+            .map { data in data.publications.compactMap { $0 } }
+            .sink { completion in
+                networkState.handleCompletion(screen: .magazinesList, completion)
+            } receiveValue: {
+                state = .results((
+                    trendingMagazines: [Magazine](),
+                    otherMagazines: [String : [Magazine]]()
+                ))
+                done()
+            }
+    }
+    
+    private var featureMagazinesSection: some View {
+        Group {
+            Header("Featured")
+                .padding([.top, .horizontal])
+            
+            ScrollView(.horizontal, showsIndicators: false) {
+                HStack(spacing: 24) {
+                    switch state {
+                    case .loading:
+                        ForEach(0..<10) { _ in
+                             MagazineCell.Skeleton()
+                        }
+                    case .reloading(let results), .results(let results):
+                        // TODO: Replace with results.trendingMagazines when backend is setup
+                        ForEach(0..<10) { _ in
+                            NavigationLink(destination: MagazineDetail()) {
+                                MagazineCell()
+                            }
+                        }
+                    }
+                }
+            }
+            .padding(.horizontal)
+        }
+    }
+
+    private var moreMagazinesSection: some View {
+        Group {
+            Header("More magazines")
+                .padding([.top, .horizontal])
+
+            ScrollView(.horizontal, showsIndicators: false) {
+                HStack(spacing: 24) {
+                    switch state {
+                    case .loading:
+                        ForEach(0..<10) { _ in
+                             MagazineCell.Skeleton()
+                        }
+                    case .reloading(let results), .results(let results):
+                        // TODO: Replace with results.trendingMagazines when backend is setup
+                        ForEach(0..<10) { _ in
+                            NavigationLink(destination: MagazineDetail()) {
+                                MagazineCell()
+                            }
+                        }
+                    }
+                }
+            }
+            .padding(.horizontal)
+        }
+    }
+    
+    var body: some View {
+        RefreshableScrollView(onRefresh: { done in
+            switch state {
+                case .loading, .reloading:
+                    return
+                case .results(let results):
+                    state = .reloading(results)
+                    fetchContent(done)
+                }
+            }) {
+                VStack {
+                    featureMagazinesSection
+                    Spacer().frame(height: 16)
```

Comment:
Tab don't work on github, so fix my spacing : )
```suggestion
                    Spacer()
                         .frame(height: 16)
```